### PR TITLE
Language: update translation de

### PR DESCRIPTION
**Pull request by hawkeyexp**
Just an update to the german lang file - added missing parts from 'en' file and translated it
Please have an eye on it for typos etc. :-)

### CODE REVIEW

File: resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -326,7 +326,7 @@ msgstr "Beim Weiterspringen pausieren"
 
 msgctxt "#30082"
 msgid "Remember audio / subtitle preferences"
-msgstr ""
+msgstr "Audio- und Untertitel-Einstellungen beibehalten"
```
Comment (Smeulf) [inline resources/language/resource.language.de_de/strings.po]: I'm not a German speaker, so please forgive my ignorance, but is "Audio- und" correct or is it a typo?

**Comment by hawkeyexp** [comment]
no problem - yes it is correct :-)
________________________________
Von: Smeulf <notifications@github.com>
Gesendet: Montag, 23. September 2019 03:11
An: CastagnaIT/plugin.video.netflix
Cc: hawkeyexp; Author
Betreff: Re: [CastagnaIT/plugin.video.netflix] Language: update translation de (#223) 

@Smeulf commented on this pull request.

________________________________

In resources/language/resource.language.de_de/strings.po:

> @@ -326,7 +326,7 @@ msgstr "Beim Weiterspringen pausieren"

 

 msgctxt "#30082"

 msgid "Remember audio / subtitle preferences"

-msgstr ""

+msgstr "Audio- und Untertitel-Einstellungen beibehalten"

I'm not a German speaker, so please forgive my ignorance, but is "Audio- und" correct or is it a typo?

File: resources/language/resource.language.de_de/strings.po (hunk 1 above)
Comment (hawkeyexp) [reply resources/language/resource.language.de_de/strings.po]: Yes it is correct - it's the short version of "Audioeinstellungen und Untertiteleinstellungen"

Comment (Smeulf) [reply resources/language/resource.language.de_de/strings.po]: Well, I learned something! Interesting syntax I couldn't imagine it exists.

File: resources/language/resource.language.de_de/strings.po [hunk 2]
```diff
@@ -559,3 +559,287 @@ msgstr "Bibliothek auf neues Format migrieren"
...
+msgid "Last used"
+msgstr "Zuletzt benutzt"
+
+msgctxt "#30163"
+msgid "Audio description"
+msgstr "Hörfilme"
+
+msgctxt "#30164"
+msgid "Browse contents with audio description."
+msgstr ""
```
Comment (CastagnaIT) [inline resources/language/resource.language.de_de/strings.po]: here missing translation

File: resources/language/resource.language.de_de/strings.po [hunk 3]
```diff
@@ -559,3 +559,287 @@ msgstr "Bibliothek auf neues Format migrieren"
...
+msgid "This device handles the auto-updates of shared-library"
+msgstr "Dieses Gerät verwaltet das Autoupdate der Datenbank"
+
+msgctxt "#30211"
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "Ein anderes Gerät verwaltet das Autoupdate der Datenbank"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "Es ist kein Gerät definiert für die Verwaltung des Autoupdates der Datenbank"
```
Comment (CastagnaIT) [inline resources/language/resource.language.de_de/strings.po]: here missing blank line at end of file

File: resources/language/resource.language.de_de/strings.po (hunk 2 above)
Comment (hawkeyexp) [reply resources/language/resource.language.de_de/strings.po]: Updated

File: resources/language/resource.language.de_de/strings.po (hunk 3 above)
Comment (hawkeyexp) [reply resources/language/resource.language.de_de/strings.po]: Added

**Pull request merged**